packages/lodestar/src/util/buffer.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Cast Uint8Array to Buffer efficiently
+ */
+export function toBuffer(bytes: Uint8Array): Buffer {
```

Comment:
```suggestion
export function toBufferUnsafe(bytes: Uint8Array): Buffer {
```
This cast also means that change on original array would be reflected on Buffer which is not what we always wan't